lambdas/search/index.py

```diff
@@ -91,6 +94,7 @@ def lambda_handler(request):
             raise ValueError("'packages' action searching indexes that don't end in '_packages'")
         _source = user_source
         size = user_size
+        es_from = user_from
```

Comment:
I think I understand why we did this, but is there ever a case where the frontend sends `from=` and we shouldn't use it? If so it's better to fail way up top, i.e. `raise ValueError("...") if from and action != 'packages'`

Reply:
last thing is let's add unit test coverage for this. should be easy to hack existing tests. i can show you next mtg. it's super constructive if you can modify the ES lambdas in general, so thanks for tackling this.

Reply:
i think it's safe to accept `from_` in all cases. now trying to fix the tests

Reply:
the tests seem ok